src/writer/serve.py

```
@@ -146,6 +155,75 @@
+        return self.api_client.get_value(
+            f"{self.default_key_prefix}{key}",
+            user_id=user_id
+            )
+
+    def delete(self, key: str, user_id: Optional[str] = None):
+        self.api_client.delete_value(
+            f"{self.default_key_prefix}{key}",
+            user_id=user_id
+            )
+
+    def put(self, key: str, value: Any, user_id: Optional[str] = None):
```

Pylint (3.3.7)

[warning] Parameter 'job_id' has been renamed to 'key' in overriding 'WriterKVJobVault.put' method

(W0237)